docs/source/development/cdf_guide.rst

```diff
@@ -0,0 +1,677 @@
+.. _cdf_guide:
+
+
+##############################
+Generating CDF Files
+##############################
+
+******************************
+CDF Files Introduction
+******************************
+CDF files are binary files which require special libraries to create and manipulate. Goddard Space Flight Center (GSFC) maintains these CDF libraries, and is the shepherd of the file format.
+
+The GSFC describes CDF files as a "self-describing data format for the storage of scalar and multidimensional data in a platform- and discipline-independent way".
+
+IMAP is archiving its science data with the `Space Physics Data Facility (SPDF) <https://spdf.gsfc.nasa.gov/>`_, and they require data to be in the CDF file format.
+
+CDF Ecosystem
+===============
+**CDF** refers to more than just the ".cdf" files themselves, but the entire toolkit surrounding those files.  The official toolkits are available in IDL, C, Fortran and MATLAB.
```

Comment:
Does CDF really refer to more than just the format itself? I'm not so sure and don't see that mentioned anywhere... Sure there are other toolkits that help with CDF files, but I think they are external to the format.

Maybe add a link to the specific CDF landing page somewhere here too: https://cdf.gsfc.nasa.gov/

Reply:
They call the CDF files themselves "dotCDF" files in the documentation, otherwise I believe technically "CDF" is the data abstraction and software interfaces/libraries that access those files.  So I guess it depends on how you think about it.  But here is a quotes from the user guide: 

"When people first hear the term CDF they intuitively think of data formats in the traditional sense of the word (i.e., messy/convoluted storage of data on disk or tape).  CDF is more than just a format." 
(skip a few lines) 
"In addition to being a self-describing data format, CDF is also a software library."